.github/workflows/release.yml

```
@@ -37,12 +35,12 @@ jobs:
       group: "release-approval"
-      cancel-in-progress: false
+      cancel-in-progress: true
     steps:
       - name: Approve release
         run: echo "Release approved in the release environment"
 
-  # Use the reusable SLSA trusted tag workflow for releases if approved
+  # Use the reusable trusted tag workflow for releases if approved
   release:
     needs: [check-release, release-approval]
     if: github.event.action != 'labeled' && needs.check-release.outputs.skip != 'true'
```

actionlint (1.7.4)

property "check-release" is not defined in object type {release-approval: {outputs: {}; result: string}}

(expression)